### ISSUE TITLE
Why the loss for discriminator is not binary_crossentropy?

### ISSUE THREAD

**Issue by victorygod** [bendangnuksung/Image-OutPainting#12]
`def dcrm_loss(y_true, y_pred):
    return -tf.reduce_mean(tf.log(tf.maximum(y_true, EPSILON)) + tf.log(tf.maximum(1. - y_pred, EPSILON)))`

Here is your dcrm_loss. However, I think the correct loss for discriminator is just binary_crossentropy, which is:

-y_true*tf.reduce_mean(tf.log(tf.maximum(y_pred, EPSILON)) + (1-y_true)*tf.log(tf.maximum(1. - y_pred, EPSILON)))
